kaznet/apps/ona/tests/test_api.py

```diff
@@ -248,10 +251,58 @@ def test_get_xform(self, mocked):
         }
 
         url = urljoin(settings.ONA_BASE_URL, 'api/v1/forms/53')
-        mocked.get(url, json=mocked_xform_data)
+        mocked.get(url, json=mocked_xform_data1)
+
+        # data returned from the 'api/v1/forms/{form_id}/form.json' endpoint
+        mocked_xform_data2 = {
```

Comment:
Can this data be updated to contain the same data as `mocked_xform_data1` since it's supposed to be basically the same form. Just that `mocked_xfrom_data2` is more updated ?

Reply:
Could we also change the naming to something else more verbose ? Like `up_to_date_form_data` or something of the sort ? The current naming makes it seem like its a completely different form.

Reply:
done!